File: Tests/installation_tests/cocoapods/with_frameworks_objc/CocoapodsTest/ViewController.m
//
//  ViewController.m
//  CocoapodsTest
//
//  Created by Ben Guo on 3/20/18.
//  Copyright © 2018 Stripe. All rights reserved.
//

#import "ViewController.h"
#import <Stripe/Stripe.h>

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    [Stripe setDefaultPublishableKey:@"foo"];
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


@end

File: Tests/installation_tests/carthage/CarthageTest/ViewController.swift
//
//  ViewController.swift
//  CarthageTest
//
//  Created by Jack Flintermann on 8/4/15.
//  Copyright (c) 2015 jflinter. All rights reserved.
//

import Stripe
import StripeApplePay
import StripeCardScan
@_spi(PrivateBetaConnect) import StripeConnect
import StripeFinancialConnections
import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        StripeAPI.defaultPublishableKey = "test"
        StripeAPI.paymentRequest(withMerchantIdentifier: "test", country: "US", currency: "USD")

        let _ = FinancialConnectionsSheet(
            financialConnectionsSessionClientSecret: "",
            returnURL: nil
        )

        let _ = CardImageVerificationSheet(
            cardImageVerificationIntentId: "foo",
            cardImageVerificationIntentSecret: "foo"
        )

        if #available(iOS 15.0, *) {
            let _ = EmbeddedComponentManager {
                return nil
            }
        }
    }
}
